peachjam/views/causelist.py

```diff
@@ -22,6 +23,8 @@
 from peachjam.views.courts import MonthMixin, RegistryMixin
 from peachjam.views.generic_views import YearMixin
 
+CACHE_DURATION = 60 * 60 * 24
```

Comment:
I think this is too high -- it means it will take a day before a new court will show up -- this call isn't THAT expensive! 30 minutes should be sufficient and  matches what we cache other pages for.

Reply:
i've set the default timeout on settings.py